scripts/copy_from_pqclean/src/sig/family/sig_family.h

```diff
@@ -9,7 +9,7 @@
 #define OQS_SIG_{{ family }}_{{ scheme['scheme'] }}_length_secret_key {{ scheme['metadata']['length-secret-key'] }}
 #define OQS_SIG_{{ family }}_{{ scheme['scheme'] }}_length_signature {{ scheme['metadata']['length-signature'] }}
 
-OQS_SIG *OQS_SIG_{{ family }}_{{ scheme['scheme'] }}_new();
+OQS_SIG *OQS_SIG_{{ family }}_{{ scheme['scheme'] }}_new(void);
```

Comment:
Can you remove the double underscores from `__OQS_SIG` in the first and second lines and re-run the templating?